dapps/mobile-optimize.md

```diff
@@ -26,20 +26,20 @@ iOS users require the latest build from Apple App Store. UI/UX design needs to a
 
 ## Master Web3
 
-Web3 is incredibly powerful for developing DApps and outlining their interactions with Ethereum or Solana nodes. The most up-to-date Web3 open-source [Ethereum JavaScript library](https://github.com/ethers-io/ethers.js/) or [Solana JavaScript library](https://solana-labs.github.io/solana-web3.js/modules.html) needs to be implemented for your DApp to be well optimized for the Trust Wallet browser.
```

Comment:
I feel like this paragraph is not relevant

Reply:
Move to a bottom "Reference" section? suggested libraries

Reply:
good idea